matscipy/calculators/manybody/potentials.py

```diff
@@ -74,3 +78,53 @@ def hessian(self, r_p, xi_p):
             np.zeros_like(xi_p),
             np.zeros_like(xi_p),
         ])
+
+class HarmonicAngle(Manybody.Theta):
+    """
+    Implementation of a harmonic angle interaction.
+    """
+
+    def __init__(self, k0=0.1, theta0=149.47):
```

Comment:
`theta0` should be in radians here. Also probably better to stick with simple values, like `k0=1`, `theta0=np.pi/2`

Reply:
You are right, I will change this